repose-aggregator/components/filters/api-validator-filter/src/main/java/org/openrepose/filters/apivalidator/ApiValidatorHandler.java

```diff
@@ -153,9 +146,16 @@ public void doFilter(HttpServletRequest request, HttpServletResponse response, F
                 }
 
                 if (!isValid) {
-                    if (mbcsInvalidRequests != null) {
+                    if (Optional.ofNullable(metricsService).isPresent()) {
```

Comment:
Do we still need the null check on the metrics service? I feel like that had to do with the way we were loading things before Spring, but with Spring, I'm not sure how it would ever have a null value.